tests/bugs/gh-3085.slang

```diff
@@ -0,0 +1,29 @@
+//TEST:SIMPLE(filecheck=CHECK): -entry MainCs -stage compute -profile glsl_450 -target spirv
+//CHECK: EntryPoint
```

Comment:
Perhaps the check could check for the desired SPIR-V op/s to have better coverage.